src/components/tabs/Tabs.module.css

```
@@ -7,19 +7,49 @@
 	margin-inline-start: -1rem;
 	overflow-x: scroll;
 	/* Hide scrollbar on firefox */
-	scrollbar-width: none; 
+	scrollbar-width: none;
 }
 
 .tab-scroll-overflow::-webkit-scrollbar {
 	/* Hide scrollbar on chrome / safari */
-  display: none;
+	display: none;
 }
 
-.tab:focus {
+.tab,
+.tab:focus,
+.tab:active {
+	border-bottom: 0 !important;
+}
+
+.tab:focus-visible {
 	outline: 2px solid var(--theme-accent);
 	outline-offset: -2px;
 }
 
 .tabpanel {
 	padding-block-start: 1rem;
 }
+
+.tablist {
+	position: relative;
+}
+
+.selectedIndicator {
+	position: absolute;
+	bottom: -4px;
```

We need a couple of extra CSS properties to prepare our indicator for the `transform` shown in the previous comment.

```suggestion
	transform-origin: left;
	left: 0;
	bottom: -4px;
```